jenkins/automation/alex.groovy

```diff
@@ -1,18 +1,12 @@
 pipeline {
     agent {
           node {
-             label "${j_agent}"
+             label "ceph-build-node"
```

Comment:
why are you using ceph node here?

Reply:
Because I am facing a lot of issues on rockylinux-8.4 and centos-7.9.2009. Tried to troubleshoot but it is failing to execute the python file and throwing weird errors.

Reply:
@gauravchaudhari02  - Please follow this conversation https://github.com/Seagate/cortx-re/pull/1064#discussion_r894130070

Reply:
okay